packages/platform-sdk-ada/src/services/message.ts

```diff
@@ -15,7 +15,7 @@ export class MessageService implements Contracts.MessageService {
 
 	public async sign(input: Contracts.MessageInput): Promise<Contracts.SignedMessage> {
 		try {
-			const privateKey = deriveRootKey(BIP39.normalize(input.mnemonic));
+			const privateKey = deriveRootKey(BIP39.normalize(input.mnemonic!));
```

Comment:
This needs a proper check that the mnemonic is set now that it isn't enforced by TypeScript.